boxoffice/views/admin_report.py

```diff
@@ -8,24 +8,25 @@
 from boxoffice.views.utils import csv_response
 from babel.dates import format_datetime
 
+
 def jsonify_report(data_dict):
-    return jsonify(org_name=data_dict['item_collection'].organization.name, item_collection_title=data_dict['item_collection'].title)
+    return jsonify(org_name=data_dict['item_collection'].organization.name, title=data_dict['item_collection'].title)
```

Comment:
Right, but since organization's `name` is being returned, this shouldn't be a problem?